game/libxml++-impl.hh

```diff
@@ -33,5 +33,19 @@ namespace xmlpp {
 	static inline void set_first_child_text(Element* element, const Glib::ustring& content) {
 		return element->set_first_child_text(content);
 	}
+#elif LIBXMLPP_VERSION_5_0
+	typedef Node::const_NodeSet const_NodeSet; // correct libxml++ 5.0 implementation
+
+	static inline Element* add_child_element(Element* element, const std::string& name) {
+		return element->add_child_element(name);
+	}
+
+	static inline const TextNode* get_first_child_text(const Element& element) {
+		return element.get_first_child_text();
+	}
+
+	static inline void set_first_child_text(Element* element, const std::string& content) {
+		return element->set_first_child_text(content);
+	}
```

Comment:
Could we avoid these wrappers by dropping support for 2.6 and then using the element accessors directly? They seem identical for 3.0 and 5.0, except for the string type.

![image](http://adrinael.net/copypasta)

Reply:
2.6 is sadly still in main debian repo's without 'easy' access to 3.0 :(

Reply:
2.6 is the default on Ubuntu 18.04 so it is difficult to remove it without breaking stuff
It seems that the versioning of xmlpp is not exactly what it seems to be... meaning 5.0 is not necessarily "better" than 2.6 or 3.0.

Reply:
Correct. 5.0 is just libxml++3 but without depending on glibmm
![image](https://user-images.githubusercontent.com/3918517/128565574-3a663845-93ff-4152-8f20-7004d88a6e1d.png)

Reply:
@Tronic I've asked around and came up with the new result